highlighters/main/test-data/precommand-killing1.zsh

```diff
@@ -30,10 +30,10 @@
 
 hash sudo=false
 
-BUFFER='sudo -e /etc/passwd'
+BUFFER='sudo -e /usr'
```

Comment:
On this value of $BUFFER, since $CURSOR isn't equal to $#BUFFER, it would be correct to flag `/usr` as `unknown-token`, even if the code doesn't do this yet.

I suppose we could `touch foo` and then use `BUFFER='sudo -e ./foo'`, couldn't we?

Reply:
`touch foo` sounds like the more natural solution, yep.

Reply:
LGTM